src/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+      integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Capstone02</title>
+  </head>
+  <body>
+    <nav class="main-nav">
+      <div class="logo-container">
+        <i class="fa fa-cutlery" aria-hidden="true"></i>
+        <h1>Best Meals</h1>
+      </div>
+      <div>
+        <ul class="nav-menu">
+          <li>Food</li>
```

Comment:
- [ ] You did good adding a comments counter to the food item's popup page. But the page is missing food items counter on the home page. Please add a counter for the total number of foods somewhere in the homepage.